hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/node/NodeManager.java

```diff
@@ -129,6 +138,18 @@
    */
   void removePipeline(Pipeline pipeline);
 
+  /**
+   * Get the entire Node2PipelineMap.
+   * @return Node2PipelineMap
+   */
+  Node2PipelineMap getNode2PipelineMap();
+
+  /**
+   * Set the Node2PipelineMap.
+   * @param node2PipelineMap Node2PipelineMap
+   */
+  void setNode2PipelineMap(Node2PipelineMap node2PipelineMap);
```

Comment:
I would suggest to remove this function from interface definition since it's only used in test.

Reply:
Updated.

Reply:
Removed from NodeManager interface.